src/commands/exile_commands.py

```diff
@@ -105,11 +109,21 @@ async def roulette(interaction: discord.Interaction):
     @bot.tree.command()
     @discord.app_commands.check(is_user_moderator)
     @discord.app_commands.describe(user="User whose exile is being checked")
-    async def check_exile(interaction: discord.Interaction, user: discord.Member):
+    async def view_exile(interaction: discord.Interaction, user: discord.Member):
         """Check the exile status of a user."""
 
         async with create_response_context(interaction) as response_message:
             async with create_logging_embed(interaction, user=user) as logging_embed:
                 msg = await get_user_exiles(logging_embed, user)
 
                 response_message.set_string(msg)
+
+    @bot.tree.command()
+    @discord.app_commands.check(is_user_moderator)
+    async def view_active_exiles(interaction: discord.Interaction):
+        """Check all active exiles."""
```

Comment:
nit can this explicitly call out that it's for all users that are exiled? Something like "Check all active exiles for all users."

Reply:
Fixed in latest